docs/cli.md

```diff
@@ -527,6 +547,30 @@ gcli tag update --tag tagA --rename newTag
 gcli tag update --tag tagA --comment "new comment"
 ```
 
+### Fileset commands
```

Comment:
This part is unrelated with this PR; is it a merge issue?

Reply:
Yes merge had an issue I'll fix it